### ISSUE TITLE
Bert_vits2模型加载问题

### ISSUE THREAD

**Issue by HankZhangZ** [Artrajz/vits-simple-api#97]
`D:\Anaconda\envs\vtuber_py\python.exe D:\HankZHanG\Cityu\EE5438\Teamwork\api\app.py 
INFO:root:Loading yaml from D:\HankZHanG\Cityu\EE5438\Teamwork\api\config.yml
2023-11-20 04:36:38 [INFO] [bert_handler.load_bert:84] Loading BERT model: D:\HankZHanG\Cityu\EE5438\Teamwork\api\bert_vits2/bert/deberta-v2-large-japanese
2023-11-20 04:36:40 [INFO] [bert_handler.load_bert:89] Success loading: D:\HankZHanG\Cityu\EE5438\Teamwork\api\bert_vits2/bert/deberta-v2-large-japanese
2023-11-20 04:36:40 [INFO] [bert_handler.load_bert:84] Loading BERT model: D:\HankZHanG\Cityu\EE5438\Teamwork\api\bert_vits2/bert/deberta-v3-large
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
D:\Anaconda\envs\vtuber_py\lib\site-packages\transformers\convert_slow_tokenizer.py:454: UserWarning: The sentencepiece tokenizer that you are converting to a fast tokenizer uses the byte fallback option which is not implemented in the fast tokenizers. In practice this means that the fast version of the tokenizer can produce unknown tokens whereas the sentencepiece version would have converted these unknown tokens into a sequence of byte tokens matching the original piece of text.
  warnings.warn(
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Some weights of the model checkpoint at D:\HankZHanG\Cityu\EE5438\Teamwork\api\bert_vits2/bert/deberta-v3-large were not used when initializing DebertaV2ForMaskedLM: ['mask_predictions.classifier.bias', 'mask_predictions.classifier.weight', 'mask_predictions.dense.weight', 'lm_predictions.lm_head.bias', 'lm_predictions.lm_head.LayerNorm.bias', 'deberta.embeddings.position_embeddings.weight', 'mask_predictions.dense.bias', 'lm_predictions.lm_head.LayerNorm.weight', 'mask_predictions.LayerNorm.bias', 'lm_predictions.lm_head.dense.weight', 'mask_predictions.LayerNorm.weight', 'lm_predictions.lm_head.dense.bias']
- This IS expected if you are initializing DebertaV2ForMaskedLM from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing DebertaV2ForMaskedLM from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
Some weights of DebertaV2ForMaskedLM were not initialized from the model checkpoint at D:\HankZHanG\Cityu\EE5438\Teamwork\api\bert_vits2/bert/deberta-v3-large and are newly initialized: ['cls.predictions.transform.dense.weight', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.bias', 'cls.predictions.transform.LayerNorm.bias']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
2023-11-20 04:36:43 [INFO] [bert_handler.load_bert:89] Success loading: D:\HankZHanG\Cityu\EE5438\Teamwork\api\bert_vits2/bert/deberta-v3-large
2023-11-20 04:36:43 [INFO] [bert_handler.load_bert:84] Loading BERT model: D:\HankZHanG\Cityu\EE5438\Teamwork\api\bert_vits2/bert/chinese-roberta-wwm-ext-large
Some weights of the model checkpoint at D:\HankZHanG\Cityu\EE5438\Teamwork\api\bert_vits2/bert/chinese-roberta-wwm-ext-large were not used when initializing BertForMaskedLM: ['cls.seq_relationship.bias', 'cls.seq_relationship.weight']
- This IS expected if you are initializing BertForMaskedLM from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertForMaskedLM from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
2023-11-20 04:36:45 [INFO] [bert_handler.load_bert:89] Success loading: D:\HankZHanG\Cityu\EE5438\Teamwork\api\bert_vits2/bert/chinese-roberta-wwm-ext-large
D:\Anaconda\envs\vtuber_py\lib\site-packages\torch\nn\utils\weight_norm.py:30: UserWarning: torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.
  warnings.warn("torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.")
2023-11-20 04:36:46 [INFO] [utils.load_checkpoint:65] Loaded checkpoint 'D:/HankZHanG/Cityu/EE5438/Teamwork/api/Model/bert_vits2/G_11000.pth' (iteration 917)
2023-11-20 04:36:46 [INFO] [ModelManager._load_model_from_path:217] model_type:BERT-VITS2 model_id:0 n_speakers:1 model_path:D:/HankZHanG/Cityu/EE5438/Teamwork/api/Model/bert_vits2/G_11000.pth
2023-11-20 04:36:46 [INFO] [ModelManager.log_device_info:142] PyTorch Version: 2.1.0+cu118 Cuda available:True Device type:cuda
2023-11-20 04:36:46 [INFO] [ModelManager.log_device_info:147] Using GPU on NVIDIA GeForce RTX 4060 Laptop GPU, GPU Device Index: None
2023-11-20 04:36:46 [INFO] [ModelManager.model_init:93] [BERT-VITS2] 1 speakers
2023-11-20 04:36:46 [INFO] [ModelManager.model_init:95] 1 speakers in total.
2023-11-20 04:36:46 [INFO] [phrases_dict.phrases_dict_init:30] Loading phrases_dict
Building prefix dict from the default dictionary ...
Loading model from cache C:\Users\HANKM~1.ZHA\AppData\Local\Temp\jieba.cache
2023-11-20 04:36:47 [WARNING] [phrases_dict.load_phrases_from_file:24] File D:\HankZHanG\Cityu\EE5438\Teamwork\api/phrases_dict.txt not found. You can create D:\HankZHanG\Cityu\EE5438\Teamwork\api/phrases_dict.txt and write your phrases_dict.
2023-11-20 04:36:47 [DEBUG] [__init__.initialize:113] Building prefix dict from the default dictionary ...
2023-11-20 04:36:47 [DEBUG] [__init__.initialize:132] Loading model from cache C:\Users\HANKM~1.ZHA\AppData\Local\Temp\jieba.cache
Loading model cost 0.379 seconds.
Prefix dict has been built successfully.
2023-11-20 04:36:47 [DEBUG] [__init__.initialize:164] Loading model cost 0.379 seconds.
2023-11-20 04:36:47 [DEBUG] [__init__.initialize:166] Prefix dict has been built successfully.
2023-11-20 04:36:47 [DEBUG] [win32._get_localzone_name:58] Looking up time zone info from registry
2023-11-20 04:36:47 [INFO] [base.start:181] Scheduler started
2023-11-20 04:36:47 [INFO] [base._real_add_job:895] Added job "clean_task" to job store "default"
 * Serving Flask app 'app'
 * Debug mode: off
2023-11-20 04:36:47 [INFO] [_internal._log:96] WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:23456
 * Running on http://192.168.1.228:23456
2023-11-20 04:36:47 [INFO] [_internal._log:96] Press CTRL+C to quit
`

以上是加载模型时候的问题，bert模型全部来自于软件自行下载，服务启动正常但是却无法加载模型，便向您请教

**Comment by Artrajz** [comment]
从日志上看，模型加载部分正常的，也加载了一个11000步的模型。可以试下语音合成是否也正常

**Comment by HankZhangZ** [comment]
重新测试后发现语音合成仍然不正常，api调用后会返回“id 0”不存在的情况

**Comment by Artrajz** [comment]
是否使用了正确的接口调用呢，你加载的模型是Bert-VITS2的
